src/services/item/ws/item.events.ts

```diff
@@ -1,7 +1,7 @@
-/**
- * Item websocket events are registered under these topics
- */
```

Comment:
The comment has been removed, was it intentional?

Reply:
Yes, I did not see the interest of having that comment on the import. Do you think we should keep it ?